src/language/lexer.js

```diff
@@ -361,6 +361,34 @@ function unexpectedCharacterMessage(code) {
   return `Cannot parse the unexpected character ${printCharCode(code)}.`;
 }
 
+function consumeNewline(
+  startPosition: number,
+  code: number,
+  codeahead: number,
+  lexer: Lexer<*>,
+): [boolean, number] {
```

Comment:
@langpavel It complicates code :(
Just revert to previous commit.
My point was that sometime in future we need to refactor Lexer, and your previous code was good until then.

Reply:
Ok, I will force push up to previous commit

Reply:
@langpavel Sorry for confusing you

Reply:
Last commit deleted